engines/bops_core/app/assets/stylesheets/bops_core/components/_header_bar.scss

```diff
@@ -0,0 +1,66 @@
+.bops-header-bar {
+  width: 100%;
+  background-color: govuk-colour("white");
+  border-bottom: 1px solid govuk-colour("mid-grey");
+}
+
+.bops-header-bar--sticky {
+  position: sticky;
+  top: var(--header-height, 0);
+  z-index: 2;
+}
+
+.bops-header-bar--static {
+  position: relative;
+  z-index: 10;
+}
```

Comment:
Stickiness should now be handled in the layout, the header bar won't need to set anything.